altvm_contracts/ton/contracts/imports/errors.fc

```diff
@@ -0,0 +1,13 @@
+const int error::unknown_op = 0xffff;
+const int error::wrong_mailbox_version = 100;
```

Comment:
I wonder why one error constant is in hex while others are in dec. Is it a convention?

Reply:
Fixed